### PR TITLE
Help how to implement a `View` to a `ViewModel` with `ISupportsActivation`.

### PR DESCRIPTION
**Pull request by fidergo-stephane-gourichon**
As discussed on slack. Any feedback welcome.

### CODE REVIEW

File: input/docs/handbook/when-activated/index.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ public class ActivatableViewModel : ISupportsActivation
 
 # View
 
+## How to ensure `ActivatableViewModel` gets activated?
```
Comment (worldbeater) [inline input/docs/handbook/when-activated/index.md]: Maybe worth mentioning what the "ActivatableViewModel" exactly is. 
A class that implements `ISupportsActivation` or so

File: input/docs/handbook/when-activated/index.md [hunk 2]
```diff
@@ -41,6 +41,12 @@ public class ActivatableViewModel : ISupportsActivation
...
+
+The framework will acknowledge the link from your `ViewModel` to your `View` if the latter implements the pattern on [Getting Started -- 4. Create Views](../../getting-started/#create-views). Watch out for `public static readonly DependencyProperty ViewModelProperty` and and the normal and explicit implementations of `IViewFor<AppViewModel>`.
```
Comment (worldbeater) [inline input/docs/handbook/when-activated/index.md]: Also worth describing that pattern in two words here

**Comment by worldbeater** [comment]
Closing this as no response

**Comment by fidergo-stephane-gourichon** [comment]
## About your feedback

Thank you for the feedback you provided when I submitted this PR. Your sentences are extremely concise and not as clear to me as they are to you.

The conventions you seem to mention are not mine, they come from the rest of the documentation/samples.

## About the PR

This PR is intended to clarify a gotcha that cost me time, so that other users don't fall into it.
I'm a little surprised you closed it, since I believe that the documentation is improved with the PR as is.
Also, this PR follows a conversation on Slack, if my memory is correct.

Isn't the documentation better (at least a tiny bit) with this PR? Had these sentences existed before in the documentation, I would not have wasted time with a non-working program + investigation. I wrote for the newcomers. The better use of the newcomers' time using ReactiveUI (and less frustration) is IMHO an improvement.

## What now?

Still I guess the documentation can be improved further with the changes you had in mind. Feel free to offer any change, or just accept the PR, in the spirit of wiki-like incremental improvement.

**Comment by worldbeater** [comment]
Seems reasonable, thank you for your feedback! I merge this then as an incremental improvement and we'll try to add more info by ourselves.

**Pull request merged**